core/includes/config.inc

```diff
@@ -357,8 +357,16 @@ function config_uninstall_config($project, $config_name = NULL) {
  *   A ConfigStorageInterface object managing the specified configuration type.
  */
 function config_get_config_storage($type = 'active') {
-  $directory = config_get_config_directory($type);
-  return new ConfigFileStorage($directory);
+  $class = settings_get('config_' . $type . '_class', 'ConfigFileStorage');
+  switch ($class) {
+    case 'ConfigDatabaseStorage':
+      $config_location = 'db:/default/' . $type . '_config';
```

Comment:
@quicksketch Should this be:
`$config_location = 'db:/default/config_' . $type;`?

Reply:
Or starting off with `db://` as discussed, but the ending of this line is what I am referencing here.

Reply:
Thanks @laryn, I've made both changes!